test/blast/test_titles.py

```diff
@@ -552,7 +552,36 @@ def testMinMatchingReads(self):
                 [
                     'gi|887699|gb|DQ37780 Cowpox virus 15',
                 ],
-                list(result.keys()))
+                list(result))
+
+    def testMaxMatchingReads(self):
+        """
+        The filter function work correctly when passed a value for
```

Comment:
The filter function *must* work correctly...

Reply:
Fixed (and in many other places) - thanks.